plugins/TrollManagement/TrollManagementTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
```

Comment:
Tests should go in a "tests" directory under the plugin. Otherwise, our test runner won't find them.

Reply:
Gotcha.